src/utils/legacy.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+export const isLegacy = Number.parseInt(window.OC?.config.version.split('.')[0] ?? '0') < 32
```

Comment:
That doesn't look future-proof, esp. if we'll make some other changes for Nextcloud 33 =)

Maybe something more specific?
```suggestion
function isLegacyFor(majorVersion: number): boolean {
	return Number.parseInt(window.OC?.config.version.split('.')[0] ?? '0') < majorVersion
}

export const isLegacy32 = isLegacyFor(32)
```

Reply:
As this is a completely internal function, I'd say it's fine to adjust it when we hav this problem.